README.md

```diff
@@ -41,6 +41,6 @@ A complete table of available flags:
 | usepost    | Use POST to send the OCSP request (default is GET)    | `./ocsp_status -post google.com:443` |
 | dir     | Write the OCSP response to a file                     | `./ocsp_status -dir=google_resp google.com:443`|
 | nostaple| Don't use the stapled OCSP response (only use with server URLs) | `./ocsp_status -nostaple google.com:443` |
-| verbose | Print information on all lints (default is only printing failed/errored lints) | `./ocsp_status -verbose google.com:443`|
+| v | Print information on all lints (default is only printing failed/errored lints) | `./ocsp_status -verbose google.com:443`|
```

Comment:
I'm not sure if it is a rule or if it is just something I noticed in the CLI tools I've used, but it seems that usually they allow a long version using two dashes and a short version using a single dash.
(i.e. `some_command --flag1 --flag2 --verbose` and `some_command --flag1 --flag2 -v` would both work and do the same thing. I'm not sure how such a thing is implemented but is that possible?